src/components/entry/Entry.styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const StyledEntry = styled.div`
+	max-width: 40rem;
+	padding: 2rem;
+	margin: 2rem;
+
+	background-color: var(--main-bg-color);
+	border: 1px solid var(--main-bg-color-full);
+	box-shadow: 4px -3px;
+
+	h3 {
+		font-size: 1.5rem;
+		color: var(--headline-little);
+	}
```

Comment:
Consider not using nested styling, as this a pretty hard anti-pattern. A good way to avoid nested styling in this case would be to develop a typography component, which you then can use for all you typography use-cases.